.github/workflows/compare-ds.yaml

```diff
@@ -108,3 +107,10 @@ jobs:
 
             </details>
           edit-mode: replace
+      - name: Delete comment in case no changes detected but there is already a comment from previous states
```

Comment:
From the commit message I understand that this is to cover cases when some one force pushes and the new commit actually triggers no changes in compare-ds. 
But this is not much clear from the step `name`.